apps/SageAccountWeb/src/style/theme.ts

```diff
@@ -128,35 +128,25 @@ const themeObject = {
     },
     body1: {
       fontWeight: 400,
-      fontSize: '20px',
-      lineHeight: '150%',
-      letterSpacing: '-0.019em',
+      fontSize: '16px',
+      lineHeight: '20px',
+      letterSpacing: '0',
       color: '#4A5056',
     },
-    body1Italic: {},
     body2: {
       fontWeight: 400,
       fontSize: '16px',
       lineHeight: '24px',
     },
-    breadcrumb1: {},
-    breadcrumb2: {},
     smallText1: {
       fontWeight: 400,
       fontSize: '14px',
       lineHeight: '21px',
       color: '#878E95',
     },
-    smallText2: {},
-    smallLink: {},
-    label: {},
-    buttonLink: {},
-    hintText: {},
-    sectionTitle: {},
-    subsectionHeader: {},
-    dataFieldKey: {},
   },
-}
-const theme: Theme = createTheme(themeObject)
+})
+
+const theme: Theme = createTheme(themeOptions)
```

Comment:
Still create a theme since we're importing it directly in a few places (instead of using `useTheme`)

Reply:
Should be able to delete this